src/thermo/Thermodynamics.h

```diff
@@ -882,8 +883,8 @@ class Thermodynamics //: public StateModelUpdateHandler
      * the surface.
      */
     void surfaceMassBalance(
-        const double *const p_Yke, const double *const p_Ykg, const double T, 
-        const double P, const double Bg, double &Bc, double &hw,
+        const double *const p_Yke, const double *const p_Ykg, const double *const p_Ycp, 
```

Comment:
This is a breaking change to the API; the version number of Mutation++ should be updated to reflect this, from 1.1.3 to 1.2, or even 2.0 (if we are following semantic versioning).